docs/installation/Publish-to-GitHub-Pages.md

```diff
@@ -0,0 +1,34 @@
+# Publish to GitHub Pages
+
+Make sure you have the project set up locally first. Follow [these](/docs/installation/local.md) instructions if you haven't already.
+
+## Steps
+* Create a new branch (say publish_gh_pages). <br>
+`git checkout -b publish_gh_pages`
+
+* Install <strong>ember-cli-github-pages</strong> addon. <br>
+`ember install:addon ember-cli-github-pages`
```

Comment:
I am not sure we can install addon like this. I mean it gave me an error couple of days ago when I tried. I would suggest replacing it with ```ember install ember-cli-github-pages```

Reply:
Yes. `install:addon` is from ember 1.x era

Reply:
@mayank8318 please update the doc to match with https://github.com/poetic/ember-cli-github-pages